recipes/dexp/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "dexp" %}
+{% set version = "2021.12.21.844" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dexp-{{ version }}.tar.gz
+  sha256: 310d63c2509fc907abdb1b29bb90baebffd838c64a9119d4211badc07830a0b6
+
+build:
+  entry_points:
+    - dexp=dexp.cli.dexp_main:cli
+    - video=dexp.cli.video_main:cli
+    - install=dexp.cli.install_main:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - arbol
+    - cachey
+    - cairocffi
+    - click ==7.1.2
+    - colorcet
+    - dask-core >=21.6.0
+    - dask-cuda >=21.6.0
+    - dask-image
+    - distributed
+    - gpustat
+    - gputil
+    - gputools
+    - joblib
+    - numba
+    - numcodecs
+    - numexpr
+    - numpy >=1.20
+    - ome-zarr
+    - platformdirs
+    - pyotf
+    - pytest
+    - python >=3.7
+    - python-blosc
+    - pyyaml
+    - scikit-image
+    - scipy
+    - seaborn
+    - tifffile
+    - zarr
+
+test:
+  imports:
+    - dexp
+  commands:
+    - pip list
+    # - pip check
+    - dexp --help
+    - video --help
+    - install --help
```

Comment:
```suggestion
    - dexp-video --help
    - dexp-install --help
```

Reply:
Pinging @JoOkuma

Reply:
I agree with this. How does it work? Should I update dexp's repo?

Reply:
I would suggest it since the names are too generic and might clash with system commands.

Thanks! Also do you agree with being added as a maintainer? (we need you to agree with a message if you do indeed agree)

Thanks for the reply!